ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -376,10 +376,13 @@ def check_backend_asds(logger, disks, backend_name):
                         # check if disk is missing
                         if disk.get('port'):
                             # put object
-                            AlbaCLI.run(command="asd-set",
-                                        named_params={'host': ip_address, 'port': str(disk.get('port')),
-                                                      'long-id': disk.get('asd_id')},
-                                        extra_params=[key, value])
+                            try:
+                                AlbaCLI.run(command="asd-set",
+                                            named_params={'host': ip_address, 'port': str(disk.get('port')),
+                                                          'long-id': disk.get('asd_id')},
+                                            extra_params=[key, value])
+                            except RuntimeError:
+                                raise ConnectionFailedException('Connection failed to disk')
```

Comment:
as discussed, you don't need a full list of errors.
I could give you one that's exhaustive today, but that's no guarantee for the future. So you just want to use our message in the exception you throw.